src/Component.ts

```diff
@@ -67,7 +67,7 @@ type RestParam<T extends EventMap, K extends EventKey<T>>
  * @ko 컴포넌트의 이벤트을 관리할 수 있게 하는 클래스
  * @alias eg.Component
  */
-class Component<T extends EventMap = EventMap> {
+class Component<T extends EventMap = {}> {
```

Comment:
@WoodNeck 

There should be no conflict with existing users.


```ts
interface DefaultEventMap {
    [key: string]: any;
}
class Component<T extends EventMap = DefaultEventMap> {}
```

Reply:
What would be the conflict? Can you show me some examples?

Reply:
@WoodNeck 
```ts
// 1
class Component<T extends EventMap = EventMap> {}
```

<img width="405" alt="스크린샷 2020-10-15 오후 9 16 57" src="https://user-images.githubusercontent.com/3433062/96122096-f5fde680-0f2b-11eb-8a79-81f3e87a1f43.png">


```ts
// 2
class Component<T extends EventMap = {}> {}
```
<img width="261" alt="스크린샷 2020-10-15 오후 9 16 34" src="https://user-images.githubusercontent.com/3433062/96122125-feeeb800-0f2b-11eb-9ff9-80fea63d0022.png">


```ts
// 3
interface DefaultEventMap {
    [key: string]: { [key: string]: any };
}
class Component<T extends EventMap = DefaultEventMap> {}
```

<img width="397" alt="스크린샷 2020-10-15 오후 9 19 37" src="https://user-images.githubusercontent.com/3433062/96122226-26de1b80-0f2c-11eb-9c9a-9b49ee0cb08f.png">

Reply:
Shouldn't we have to force them to use event type definitions?
I think there's no meaning in leaving them as they are.
As the actual "users" you say is basically us, we should update our components types when we update @egjs/component dependency.

Reply:
If you use a type, you can force it.

However, all events were previously allowed. This should be the default. This is not a bug.

If forced, the interface with the existing one will be changed. Shouldn't the major version be upgraded?

Also, we will use it mainly, but we are not the only existing users.

Reply:
Okay, I got what you're saying. I doubt whether there is anyone using this component though.
I slightly changed that `DefaultEventMap` to make previous components can trigger without any props, so please check that.